src/models/ontology_models.py

```diff
@@ -0,0 +1,118 @@
+
+from sentence_transformers import util
+import pandas as pd
+import numpy as np
+from sentence_transformers import SentenceTransformer
+from transformers import AutoTokenizer, AutoModel, AutoModelForMaskedLM
+from tqdm.auto import tqdm
+import torch
+from abc import ABC, abstractmethod
```

Comment:
I would remove unused imports

Reply:
Removed these unused imports in the latest commit.